calico-enterprise/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -31,6 +31,7 @@ eBPF (or "extended Berkeley Packet Filter"), is a technology that allows safe mi
 - [Verify that your cluster is ready for eBPF mode](#verify-that-your-cluster-is-ready-for-ebpf-mode)
 - [Configure {{prodname}} to talk directly to the API server](#configure-calico-enterprise-to-talk-directly-to-the-api-server)
 - [Configure kube-proxy](#configure-kube-proxy)
+- [Disable host conntrack bypass in MKE](#disable-host-conntrack-bypass-in-mke)
```

Comment:
```suggestion
- [(MKE only) Disable host conntrack bypass](#disable-host-conntrack-bypass-in-mke)
```
I find it best to put the conditional part of the sentence up front so the user can skip ahead if this doesn't apply them.  Otherwise they're likely to read the first half of the sentence and start doing it before they reach the "but only if you're on MKE" part